api/src/main/java/io/opentelemetry/trace/propagation/ContextUtils.java

```diff
@@ -0,0 +1,179 @@
+    return context.withValue(CONTEXT_SPAN_KEY, span);
+  }
+
+  /**
+   * Creates a new {@code Context} with the given {@code SpanContext} set.
+   *
+   * @param spanContext the value to be set.
+   * @return a new context with the given value set.
+   * @since 0.3.0
+   */
+  public static Context withSpanContext(SpanContext spanContext) {
+    return withSpanContext(spanContext, Context.current());
+  }
+
+  /**
+   * Creates a new {@code Context} with the given {@code SpanContext} set.
+   *
+   * @param spanContext the value to be set.
+   * @param context the parent {@code Context}.
+   * @return a new context with the given value set.
+   * @since 0.3.0
+   */
+  public static Context withSpanContext(SpanContext spanContext, Context context) {
+    return context.withValue(CONTEXT_SPANCONTEXT_KEY, spanContext);
+  }
+
+  /**
+   * Returns the {@code Span} from the current {@code Context}.
+   *
+   * @return the value from the current {@code Context}.
+   * @since 0.1.0
+   */
+  public static Span getSpan() {
+    return CONTEXT_SPAN_KEY.get();
+  }
+
+  /**
+   * Returns the {@code Span} from the specified {@code Context}.
+   *
+   * @param context the specified {@code Context}.
+   * @return the value from the specified {@code Context}.
+   * @since 0.1.0
+   */
+  public static Span getSpan(Context context) {
+    return CONTEXT_SPAN_KEY.get(context);
+  }
+
+  /**
+   * Returns the {@code Span} from the specified {@code Context}, falling back to a default, no-op
+   * {@code Span}.
+   *
+   * @param context the specified {@code Context}.
+   * @return the value from the specified {@code Context}.
+   * @since 0.3.0
+   */
+  public static Span getSpanWithDefault(Context context) {
+    Span span = CONTEXT_SPAN_KEY.get(context);
+    return span == null ? DEFAULT_SPAN : span;
+  }
+
+  /**
+   * Returns the {@link SpanContext} from the current {@code Context}.
+   *
+   * @return the value from the current {@code Context}.
+   * @since 0.3.0
+   */
+  public static SpanContext getSpanContext() {
+    return CONTEXT_SPANCONTEXT_KEY.get();
+  }
+
+  /**
+   * Returns the {@link SpanContext} from the specified {@code Context}.
+   *
+   * @param context the specified {@code Context}.
+   * @return the value from the specified {@code Context}.
+   * @since 0.3.0
+   */
+  public static SpanContext getSpanContext(Context context) {
+    return CONTEXT_SPANCONTEXT_KEY.get(context);
+  }
+
+  /**
+   * Returns any {@link SpanContext} from the specified {@code Context}.
+   *
+   * <p>This method tries to get any non-null {@link SpanContext} in {@code Context}, giving higher
+   * priority to {@code Span#getContext()} and then falling back to {@code SpanContext}. If none is
+   * found, this method returns {@code null}.
+   *
+   * @param context the specified {@code Context}.
+   * @return the value from the specified {@code Context}.
+   * @since 0.3.0
+   */
+  public static SpanContext getAnySpanContext(Context context) {
```

Comment:
```suggestion
  public static SpanContext getEffectiveSpanContext(Context context) {
```

"Any" just sounds strange.

Reply:
I like `getEffective` more than `getAny`, so lets go with that for now.